src/Codeception/Lib/Parser.php

```diff
@@ -170,6 +170,9 @@ public static function getClassesFromFile($file)
             }
         }
 
+        $tokens = null;
+        gc_mem_caches();
```

Comment:
Please use`if (PHP_MAJOR_VERSION > 5)` to make this code work on PHP 5.6

Reply:
Done.